### PR TITLE
geth: switch to nftables

### PR DESCRIPTION
**Pull request by mendelskiv93**
- [x] [nimbus-geth-holesky]
holesky-01.ih-eu-mda1.nimbus.geth
holesky-02.ih-eu-mda1.nimbus.geth

- [x] [nimbus-geth-mainnet]
mainnet-01.aws-eu-central-1a.nimbus.geth

Referenced issue: https://github.com/status-im/infra-misc/issues/301

### CODE REVIEW

File: ansible/group_vars/nimbus-geth-mainnet.yml [hunk 1]
```diff
@@ -31,15 +31,24 @@ geth_websocket_addr: '0.0.0.0'
...
-  - { port: '{{ geth_authrpc_port }}',         ipset: 'eth2.prod' }
-  - { port: '{{ geth_websocket_port }}',       ipset: 'nimbus.mainnet' }
-  - { port: '{{ geth_websocket_port }}',       ipset: 'eth2.prod' }
+  geth-node:
+  - { port: '{{ geth_port }}',           comment: 'Geth LibP2P',    protocol: 'tcp'                         }
+  - { port: '{{ geth_port }}',           comment: 'Geth Discovery', protocol: 'udp'                         }
+  - { port: '{{ geth_expo_cont_port }}', comment: 'Geth Exporter',  ipset: 'metrics.hq',     iifname: 'wg0' }
+  - { port: '{{ geth_rpc_port }}',       comment: 'Geth RPC',       ipset: 'nimbus.mainnet', iifname: 'wg0' }
+  - { port: '{{ geth_rpc_port }}',       comment: 'Geth RPC',       ipset: 'eth2.prod',      iifname: 'wg0' }
+  - { port: '{{ geth_rpc_port }}',       comment: 'Geth RPC',       ipset: 'rocket.prod',    iifname: 'wg0' }
```
Comment (mendelskiv93) [inline ansible/group_vars/nimbus-geth-mainnet.yml:46]: ports for `rocket.prod` were opened but not in `open_ports_list`. I added it but I guess that was manually added and we don't need it?

File: ansible/group_vars/nimbus-geth-mainnet.yml
Comment (mendelskiv93) [inline ansible/group_vars/nimbus-geth-mainnet.yml:1]: please check the comments, not sure this time

File: ansible/group_vars/nimbus-geth-holesky.yml [hunk 2]
```diff
@@ -31,10 +31,14 @@ geth_expo_source_cont_name: '{{ geth_cont_name }}'
...
 # Open Ports
-open_ports_default_comment: 'Geth API'
-open_ports_default_chain: 'VPN'
 open_ports_list:
-  - { port: '9400:9404', ipset: 'metrics.hq', comment: 'Geth Metrics' }
-  - { port: '8551:8554', ipset: 'nimbus.holesky', comment: Geth }
-  - { port: '{{ smart_metrics_listen_port }}', ipset: 'metrics.hq', comment: 'SMART Metrics' }
+  geth-node:
+    - { port: '9401-9404', comment: 'Geth Exporter', ipset: 'metrics.hq',     iifname: 'wg0' }
+    - { port: '8552-8555', comment: 'Geth',          ipset: 'nimbus.holesky', iifname: 'wg0' }
```
Comment (mendelskiv93) [inline ansible/group_vars/nimbus-geth-holesky.yml:42]: port `8555` was closed but needed for `geth-holesky-04-node`:
```
services:
  geth:
    container_name: 'geth-holesky-04-node'
    image: 'ethereum/client-go:v1.14.2'
    user: root
    restart: 'always'
    stop_grace_period: '1m'
    deploy:
      resources:
        limits:
          memory: '51329m'
    ports:
      - '30307:30307/tcp'
      - '30307:30307/udp'
      - '6064:6064/tcp'
      - '127.0.0.1:8549:8549/tcp'
      - '0.0.0.0:8555:8555/tcp'
```

File: ansible/group_vars/nimbus-geth-mainnet.yml [hunk 3]
```diff
@@ -31,15 +31,24 @@ geth_websocket_addr: '0.0.0.0'
...
+  - { port: '{{ geth_authrpc_port }}',   comment: 'Geth API',       ipset: 'nimbus.mainnet', iifname: 'wg0' }
+  - { port: '{{ geth_authrpc_port }}',   comment: 'Geth API',       ipset: 'eth2.prod',      iifname: 'wg0' }
+  - { port: '{{ geth_websocket_port }}', comment: 'Geth Websocket', ipset: 'nimbus.mainnet', iifname: 'wg0' }
+  - { port: '{{ geth_websocket_port }}', comment: 'Geth Websocket', ipset: 'eth2.prod',      iifname: 'wg0' }
+  - { port: '{{ geth_websocket_port }}', comment: 'Geth Websocket', ipset: 'rocket.prod',    iifname: 'wg0' }
```
Comment (jakubgs) [inline ansible/group_vars/nimbus-geth-mainnet.yml:51]: We can drop this too. Same for `eth2.prod`.

**Review by jakubgs — APPROVED** [review]

**Review by yakimant — APPROVED** [review]

**Pull request merged**